### DISCUSSION TITLE
Unable to verify my documents

### DISCUSSION THREAD

**Discussion by iamparas0** [community/community#57332] [GitHub Education]
### Hi everyone,

I have uploaded documents but ,it can't verify my GitHub student developer pack now i getting tired of this please help me i have a limited time to submit my project and need the GitHub.

Please contact at pearsparas@gmail.com

**Comment by samus-aran** [comment]
Hi @iamparas0 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
